src/pages/JournalEntryFormPage/JournalEntryFormPage.tsx

```diff
@@ -23,6 +23,7 @@ const JournalEntryFormPage: React.FC = () => {
     const user = useContext(AuthContext);
     const navigate = useNavigate();
     const { entryId } = useParams();
+    const [entryUserId, setEntryUserId] = useState<string | null>(null)
```

Comment:
Will probably be able to get rid of this once the backend is set up. There will be a hook that retrieves the entry data from the useParams hook results. Data will then need to be used to set the form states.